package.json

```diff
@@ -3,15 +3,32 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "dotenv": "^16.0.0",
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.2.0",
     "@testing-library/user-event": "^13.5.0",
+    "@babel/eslint-parser": "^7.16.5",
+    "@babel/register": "^7.17.7",
+    "@babel/runtime": "^7.17.9",
+    "clean-webpack-plugin": "^4.0.0",
+    "copy-webpack-plugin": "^11.0.0",
+    "css-minimizer-webpack-plugin": "^4.0.0",
+    "eslint-webpack-plugin": "^3.1.1",
+    "html-webpack-plugin": "^5.5.0",
+    "mini-css-extract-plugin": "^2.6.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
     "react-scripts": "5.0.1",
```

Comment:
Consider cleaning up the project a bit to get rid of anything CRA-related.
That includes: scripts, libraries, redundant default files (like workers, vitals and tests) that come with CRA installation.

Reply:
Granted, the packages I'm referring to might be used by some "yarn" magic but that's just a byproduct of using third-party tools, eh? ;)

Reply:
By any means necessary ;)
I would've probably just removed node_modules and iterated through "dependencies" and "devDependencies" to make sure that I'm using each and every library and there are no dangling dependency.
You can quickly `npx create-react-app ...` in a separate location on you machine and just compare package.jsons

Reply:
went through all packages and remove the ones, that I don't use at all